packages/react-ui-core/src/Ratings/RatingItem.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export default class RatingItem extends Component {
+  static propTypes= {
+    theme: PropTypes.object,
+  }
+  static defaultProps = {
+    theme: {},
+  }
+
+  render() {
+    const {
+      width,
+      index,
+      backgroundColor,
+      theme,
+    } = this.props
```

Comment:
Thee props here doesn't exist in `propTypes` with the exception of `theme`. Please add the rest.